### PR TITLE
Highfive shouldn't notify if author is a watcher

### PR DESCRIPTION
**Pull request by Wafflespeanut**
It's annoying when highfive notifies the issue/PR author (when he's a label watcher)

### CODE REVIEW

**Comment by Wafflespeanut** [comment]
Oh, it's assigning to myself? I really thought I'd fixed it (meh)

File: handlers/label_watchers/tests/not_watched.json [hunk 1]
```diff
@@ -18,7 +18,10 @@
     }, 
     "action": "labeled", 
     "issue": {
-      "number": 84
+      "number": 84,
+      "user": {
+        "login": "jdm"
```
Comment (jdm) [inline handlers/label_watchers/tests/not_watched.json]: This should be a different user to ensure the test is testing what we want, I think.

File: handlers/label_watchers/tests/label_sender.json [hunk 2]
```diff
@@ -18,7 +18,10 @@
     }, 
     "action": "labeled", 
     "issue": {
-      "number": 84
+      "number": 84,
+      "user": {
+        "login": "jdm"
```
Comment (jdm) [inline handlers/label_watchers/tests/label_sender.json]: This should be a different user.

**Comment by jdm** [comment]
Looks good apart from the test comments.

**Comment by Wafflespeanut** [comment]
Yep, makes sense. You have a keen eye really! :)

**Comment by jdm** [comment]
Thanks!

**Comment by Wafflespeanut** [comment]
Thank you :)

**Pull request merged**